### DISCUSSION TITLE
[Idea] 支持创建群聊或agent组，引入多个AI agent协同完成一项任务

### DISCUSSION THREAD

**Discussion by TonyBlur** [lobehub/lobehub#1043] [Ideas]
### 🥰 需求描述

通过创建群聊或agent组，借助不同的agent分工完成特定任务、进行自我反思，最后可以更高效的得到想要的结果和成果

### 🧐 解决方案

支持将多个AI agent加入同一场对话，构成群组，分配不同的话语权优先权重，并可以相互提问调用完成协同工作

### 📝 补充信息

思路来源：【我开了一家员工全是AI的公司，自动帮我炒股-哔哩哔哩】 https://b23.tv/VjkEq86

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
今年会做的。

[a comment by an automated account is not shown]

**Comment by MapleEve** [comment]
Dify.AI 目前比较火的是这样的node块链接类，我估计等成熟之后应该也和comfy一样了，可以都观察下

**Reply by rocklau** [reply]
不灵活
